docs/tutorials/platform_beginner/6_enabling_account_registration.md

```diff
@@ -62,3 +63,70 @@ Log in the back office as an Administrator and go to the Users page. Select the
 
 Then log in again into the back office with your new user credentials.
 You have now the possibility to create new Rides and Points of Interest only.
+
+
+
+##  Customize the use forms
+
+We want to have a consistent design over our web site. When using the register form, it was a raw HTML form without any styling on it. We want to welcome our new users with a pretty register page.
+
+### Use a views.yml file
+
+In order to be clearer, we will extract the YAML directives regarding the presentation and put them into their own separate file. Create the views.yml file and cut-paste into it the parts of ezplatform.yml that are about the templating.
```

Comment:
Maybe replace `to be clearer` with something like `to have a better file structure` or `to organize our configuration better`

I think "directives" actually refer to something different in Yaml (http://yaml.org/spec/1.1/#id864824), plus using `key` or even simply `line` here would be easier to understand for a beginner.

Also, not clear about the last sentence, Does it mean we remove this part of the config from ezplatform.yml and place it in views.yml? Then I'd formulate it like this: `Create a views.yml file and move the following code referring to templating to this file from ezplatform.yml:`